### ISSUE TITLE
java.lang.NoSuchFieldError: pollenParticles

### ISSUE THREAD

**Issue by aprock** [miyo6032/bosses-of-mass-destruction#116]
We are seeing a crash in BOMD when planting void lillys using 1.20.1 fabric.

```
[17:59:15] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoSuchFieldError: pollenParticles
	at net.barribob.boss.block.VoidLilyBlockEntity.<clinit>(VoidLilyBlockEntity.kt:75) ~[BOMD-1.7.4-1.20.1.jar:?]
	at net.barribob.boss.block.ModBlocks.voidLilyBlockEntityFactory$lambda$9(ModBlocks.kt:72) ~[BOMD-1.7.4-1.20.1.jar:?]
	at net.barribob.boss.block.VoidLilyBlock.method_10123(VoidLilyBlock.kt:23) ~[BOMD-1.7.4-1.20.1.jar:?]
```

installed mods.

```
architectury-9.1.12-fabric.jar
awesomedungeonend-fabric-1.20.1-3.1.1.jar
awesomedungeon-fabric-1.20.1-3.2.0.jar
awesomedungeonnether-fabric-1.20.1-3.1.1.jar
awesomedungeonocean-fabric-1.20.1-3.3.0.jar
balm-fabric-1.20.1-7.2.1.jar
BOMD-1.7.4-1.20.1.jar
brutalbosses-fabric-1.20.1-6.6.jar
bygonenether-1.3.2-1.20.x.jar
cardinal-components-api-5.2.2.jar
cloth-config-11.1.106-fabric.jar
Clumps-fabric-1.20.1-12.0.0.3.jar
cupboard-fabric-1.20.1-2.2.jar
dragonfight-fabric-1.20.1-4.1.jar
dungeons-and-taverns-3.0.3.jar
DungeonsArise-1.20.1-2.1.57-fabric-release.jar
dungeonz-1.0.3.jar
eldritch-mobs-1.15.0.jar
extrakeybinds-0.5.0+1.17-1.19.4.jar
fabric-api-0.91.0+1.20.1.jar
fabric-language-kotlin-1.10.9+kotlin.1.9.0.jar
ForgeConfigAPIPort-v8.0.0-1.20.1-Fabric.jar
geckolib-fabric-1.20.1-4.2.4.jar
IllagerInvasion-v8.0.3-1.20.1-Fabric.jar
Incendium_1.20.1_v5.3.1.jar
libraryferret-fabric-1.20.1-4.0.0.jar
majrusz-library-fabric-1.20.1-7.0.2.jar
majruszs-enchantments-fabric-1.20.1-1.10.5.jar
malilib-fabric-1.20.1-0.16.3.jar
modmenu-7.2.1.jar
mvs-4.1.1-1.20-fabric.jar
orb_of_origins_plus_1_20_1.jar
Origins-1.20.1-1.10.0.jar
Origins-Classes-1.20-1.7.0.jar
originstweaks-1.15.4.jar
owo-lib-0.11.2+1.20.jar
pal-1.8.0.jar
Patchouli-1.20.1-83-FABRIC.jar
Pehkui-3.7.11+1.14.4-1.20.2.jar
Philips-Ruins1.20.1-2.8[Fabric].jar
player-animation-lib-fabric-1.0.2-rc1+1.20.jar
puffish_skills-0.10.0-1.20-fabric.jar
PuzzlesLib-v8.1.11-1.20.1-Fabric.jar
resourcefulconfig-fabric-1.20.1-2.1.0.jar
resourcefullib-fabric-1.20.1-2.1.19.jar
spell_power-fabric-0.9.19+1.20.1.jar
Terralith_1.20.1_v2.4.5.jar
tlc-fabric-1.0.1-1.20.X.jar
UniversalEnchants-v8.0.0-1.20.1-Fabric.jar
waystones-fabric-1.20-14.1.3.jar
yet-another-config-lib-fabric-3.2.1+1.20.jar
```

**Comment by miyo6032** [comment]
1.7.5 should fix this

**Comment by LemonDemolition** [comment]
Is there any way to fix this issue on 1.20.2 fabric?

**Comment by miyo6032** [comment]
uploaded a version that should fix for 1.20.2
